lambda-utils/lib/lambda-utils.ts

```diff
@@ -8,6 +8,31 @@ const logger = new Logger('lambda-utils');
 /** Define the async version of ProxyHandler */
 export type AsyncProxyHandler = (event: APIGatewayEvent, context: Context) => Promise<any>;
 
+/**
+ * Casted interface for APIGatewayProxyEvents converted through the middleware
+ *
+ * @typedef {Interface}
+ */
+export interface ISailplaneAPIGatewayProxyEvent extends APIGatewayProxyEvent {
+    /**
+     * HTTP Request body, parsed from a JSON string into an object.
+     * @property body
+     */
+    body: any;
+
+    /**
+     * HTTP Path Parameters, parsed from a JSON string into an object
+     * @property pathParameters
+     */
+    pathParameters: { [name: string]: string };
+
+    /**
+     * HTTP URL query string parameters, parsed from a JSON string into an object
+     * @property pathParameters
```

Comment:
Please explain to me the value of this tag. Just seems like an opportunity to be out of sync with the actual code. (Which it is indeed right here.)

Reply:
I'm not sure what you want me to explain on this one. It's a standard jsdoc element. If you'd prefer it removed I'll happily do so.

Reply:
https://jsdoc.app/tags-property.html

Reply:
From that page "This tag is intended for simple collections of static properties". See the example. This is not what that tag is for.

Reply:
Noted, and already addressed